solutions/swb-reference/integration-tests/config/example.yaml

```diff
@@ -15,6 +15,9 @@
 #rootUserNameParamStorePath: ''
 #rootPasswordParamStorePath: ''
 
+## Default Hosting Account
+#defaultHostingAccountId: ''     # exp: acc-2272f4a8-1791-419a-8e49-b926f3eb46f9
```

Comment:
This is needed because when we create a CostCenter, we need to pass in a valid `hostingAccountId`